.capirc

```diff
@@ -0,0 +1 @@
+cluster-api:v1.1.5
```

Comment:
Why prefix this file with a dot? We want it to be visible. Also, missing a newline.

Reply:
I had the exact thought- it's not code so it can be "invisible" 😅 
I'll make the change